src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -0,0 +1,398 @@
+		 *
+		 * @param {Array} crumbs The array of all crumbs
+		 * @param {Array} newCrumbs The array of the crumbs to hide and add
+		 */
+		addCrumbs(crumbs, newCrumbs) {
+			newCrumbs.forEach((crumb, i) => {
+				if (crumb.elm && crumb.elm.classList) {
+					if (this.hiddenCrumbs.includes(crumb)) {
+						crumb.elm.classList.add('hidden')
+					} else {
+						crumb.elm.classList.remove('hidden')
+					}
+				}
+				crumbs.push(crumb)
+			})
+		},
+	},
+	render: function(createElement) {
+		// Get the breadcrumbs
+		const breadcrumbs = this.$slots.default || []
+
+		// Check that we have at least one breadcrumb
+		if (breadcrumbs.length === 0) {
+			return
+		}
+
+		// Add the root icon to the first breadcrumb
+		Vue.set(breadcrumbs[0].componentOptions.propsData, 'icon', this.rootIcon)
+
+		const crumbs = []
+		// The first half of the breadcrumbs
+		const crumbs1 = this.hiddenCrumbs.length
+			? breadcrumbs.slice(0, Math.round(breadcrumbs.length / 2))
+			: breadcrumbs
+		this.addCrumbs(crumbs, crumbs1)
+
+		// The Actions menu
+		if (this.hiddenCrumbs.length) {
+			crumbs.push(createElement('div', { class: 'crumb' },
+				[createElement('Actions', {
+					class: 'dropdown',
+					props: {
+						forceMenu: true,
+					},
+					ref: 'actions',
+					nativeOn: {
+						dragstart: this.dragStart,
+						dragenter: () => { this.$refs.actions.opened = true },
+						dragleave: this.closeActions,
+					},
+				}, this.hiddenCrumbs.map(crumb => {
+					const to = crumb.componentOptions.propsData.to
+					const href = crumb.componentOptions.propsData.href
+					const disabled = crumb.componentOptions.propsData.disableDrop
+					let element = 'ActionLink'
+					let path = href
+					if (to) {
+						element = 'ActionRouter'
+						path = to
+					}
+					return createElement(element, {
+						class: 'crumb',
+						props: {
+							to: to,
+							href: href,
+							icon: 'icon-folder',
+						},
+						attrs: {
+							draggable: false,
+						},
+						nativeOn: {
+							dragstart: this.dragStart,
+							drop: ($event) => this.dropped($event, path, disabled),
+							dragover: this.dragOver,
+							dragenter: ($event) => this.dragEnter($event, disabled),
+							dragleave: ($event) => this.dragLeave($event, disabled),
+						},
+					},
+					crumb.componentOptions.propsData.title
+					)
+				}))]
+			))
+		}
+		// The second half of the breadcrumbs
+		const crumbs2 = this.hiddenCrumbs.length
+			? breadcrumbs.slice(Math.round(breadcrumbs.length / 2))
+			: []
+		this.addCrumbs(crumbs, crumbs2)
+
+		return createElement('div', { class: 'breadcrumb', ref: 'container' }, crumbs)
+	},
+}
+</script>
+
+<style lang="scss" scoped>
+@import '../../fonts/scss/iconfont-vue';
+
+.breadcrumb {
+	width: 100%;
+	flex-grow: 1;
+
+	div.crumb {
+		@include iconfont('breadcrumb');
+		background-image: none;
+		display: inline-flex;
+		height: $clickable-area;
+		padding: 0;
+
+		&::before {
+			display: flex;
+			align-items: center;
```

Comment:
you should not need the display flex. It will have no effect here, it only contains text

Reply:
It is needed to align the breadcrumb icon correctly. Without this style it looks like this:
![Screenshot_2020-03-28 Inventar - Nextcloud(1)](https://user-images.githubusercontent.com/2496460/77822818-69f2b180-70f6-11ea-8263-c863edc18e07.png)

Reply:
interesting, I would have use line-height then as this is text. I did not thought flex would align it :thinking:

Reply:
Looks good then! Let's leave it like that!

Reply:
`line-height: 44px` also works. I have no preference here. Shall we go for line-height then?

Reply:
> 
> 
> Looks good then! Let's leave it like that!

Alright.

Reply:
Let me check with ie

Reply:
All good in ie!
![image](https://user-images.githubusercontent.com/14975046/77823759-c4434080-70fd-11ea-998b-8d62997c90b7.png)